### DISCUSSION TITLE
Show all folders of current folder, after typing "cd .", when PredictionViewStyle is ListView

### DISCUSSION THREAD

**Discussion by iksi4prs** [lzybkr/PSReadLine#3634] [Q&A] [answered]
Hi,
When I type "cd ." I see that all options in list view are from history.
So some of the options are not relevant per current directory.
Is the there a way to see all folder of current directory (instead of using TAB in InlineView) ?

Or that I need to implement my own predictor as explained here?
https://learn.microsoft.com/en-us/powershell/scripting/dev-cross-plat/create-cmdline-predictor?view=powershell-7.3

Or maybe some packages/module already available ?

Thanks
Si.

Using:
PowerShell 7.3.3

**Comment by daxian-dbw** [comment, marked as answer]
It sounds https://github.com/Ink230/DirectoryPredictor is what you are looking for. You can get it from PowerShellGellary: https://www.powershellgallery.com/packages/DirectoryPredictor/0.0.4.

**Comment by ZhujinLi** [comment]
I would suggest another plugin [CompletionPredictor](https://github.com/PowerShell/CompletionPredictor), which is more reliable and handier in my opinion.
